src/apdu_sign.c

```diff
@@ -422,8 +422,10 @@ void parse_operation(struct maybe_transaction* _U_ dest, bip32_path_t* _U_ key_d
 
 	  if(!isSelf && !isDao) {
             sent_amounts+=capacity_val;
+	    // Destination address must end up the one we expect. If it's unset,
```

Comment:
This reads as if there is a particular destination we are expecting; we only care that there is only one, not what it is.